lib/get_parser.py

```diff
@@ -437,4 +437,10 @@ def get_condor_epilog() -> str:
     )
     epilog_l[0] += "(with single '-' or double '--' dashes)\n"
 
+    if condor_cmd == "condor_q":
```

Comment:
Clever!  Could you please add a comment here saying why we need to do this?  I'm afraid that six months from now, we'll have no idea why we had to replace part of the condor epilog.